shared/pdf_viewer.css

```diff
@@ -24,23 +24,18 @@
   line-height: 1.0;
 }
 
-.textLayer > div {
+.textLayer > span {
```

Comment:
Again, adjustment that would remove the `transform-origin` from the `.textLayer` div I think.